anheumai-dparys/mocking/mocking.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
+)
+
+const (
+	finalWord      = "Go!"
+	countdownStart = 3
+	write          = "write"
```

Comment:
just another usage - i would prefer write and sleep consts into test.go since they are being used there and string are immutable.

Reply:
yeah, you're right makes sense